src/store/task/types/taskTypes.js

```diff
@@ -1,4 +1,5 @@
 export const taskActionTypes = {
   ADD_TASK: "ADD_TASK",
   DELETE_TASK: "DELETE_TASK",
+  DONE_TASK: "DONE_TASK",
```

Comment:
`Done_task` is something feeling awd, can you find something more meaningful? for example: `Complete_Task`?

Reply:
@iud-wtag ditto